link/src/main/java/com/stripe/android/link/ui/LinkAppBar.kt

```diff
@@ -107,3 +100,59 @@ internal fun LinkAppBar(
         Spacer(modifier = Modifier.width(MinimumTouchTargetSize))
     }
 }
+
+@Preview
+@Composable
+internal fun LinkAppBar() {
```

Comment:
Can these be private instead of internal?

Reply:
Yep, changed!